API.md

```diff
@@ -39,6 +42,30 @@ See also [`reject`](#reject), [`resolve`](#resolve).
 all([ Promise.resolve('a') ]) //=> Promise ['a']
 ```
 
+### assign
+
+```haskell
+// assign : String -> ({ k: v } -> a) -> { k: v }
+```
+
+Accepts a property, function and object.  Sets the property on the object to the result of the function.
+
+```js
+assign('foo', {}, always('bar')) //=> { foo: 'bar' }
+```
+
+### assignP
+
+```haskell
+// assignP : String -> ({ k: v } -> Promise a) -> { k: v }
+```
+
+Accepts a property, promise returning function and object.  Sets the property on the object to the result of the function when it resolves.
+
+```js
+assignP('foo', {}, always(Promise.resolve('bar'))) //=> Promise { foo: 'bar' }
```

Comment:
This example is wrong.  `assignP` takes a Promise-returning function as the second arg, and a plain object as the third.